styles/components/_search.scss

```
@@ -91,6 +91,24 @@
 .search-bar-container {
   position: relative;
 
+  .search-bar-collection-search {
+    z-index: 2;
+    position: absolute;
+    left: 2.5rem;
+    top: 0.5rem;
+    height: 1.5rem;
+    gap: 0.35rem;
+    cursor: default;
+    span {
+      height: 21px;
+      padding: 0 14px;
```

please use rem here too

```suggestion
      height: 1.25rem;
      padding: 0 1rem;
```